src/content/docs/about/trademark.md

```diff
@@ -0,0 +1,117 @@
+
+- The main focus of the group is our software
+- The group does not make a profit
+- Any charge to attend meetings are to cover the cost of the venue, food and drink only
+
+You are not authorized to conduct a conference using our marks.
+
+### No Domain Names
+
+You must not register any domain that includes our word marks or any variant or combination of them.
+
+### Other For Profit Usage of the TAURI Marks
+
+If you are making a video, tutorial series, book, or other educational material, for which you are receiving payment through subscriptions, sales, advertising or the like, then you must acquire explicit licensing permission from The Tauri Programme within the Commons Conservancy.
+
+## How to Display Our Marks
+
+When you have the right to use our mark, here is how to display it.
+
+### Trademark Marking and Legends
+
+The first or most prominent mention of a mark on a webpage, document, or documentation should be accompanied by a symbol indicating whether the mark is a registered trademark (“®”) or an unregistered trademark (“™”). If you don’t know which applies, contact us. (TAURI itself is a registered trademark.)
+
+Place the following notice at the foot of the page where you have used the mark: TAURI is trademark of [The Tauri Programme within the Commons Conservancy].”
+
+### Use of Trademarks in Text
+
+Always use trademarks in their exact form with the correct spelling, neither abbreviated, hyphenated, or combined with any other word or words.
+
+- Unacceptable: TAUREE
+- Acceptable: TAURI
+
+Don’t pluralize a trademark.
+
+- Unacceptable: I have seventeen TAURIs running on my computer.
+- Acceptable: I am running seventeen TAURI applications on my computer and have ram to spare.
+
+Always use a trademark as an adjective modifying a noun.
+
+- Unacceptable: This is a TAURI.
+- Acceptable: This is a TAURI software application.
+
+### Use of Logos
+
+You may not change any logo except to scale it. This means you may not add decorative elements, change the colors, change the proportions, distort it, add elements, or combine it with other logos.
+
+We have a high-contrast version of the logo, which you can download below in the assets section.
+
+## Assets
+
+- Here you may download the entire [Brand Guidelines](/assets/brand_guidelines.pdf) - (PDF, 74.3 MB)
+- Here you may download SVG and PNG formats of the LOGO and Wordmark in the [Logopack](/assets/logopack.zip) - (ZIP, 203 KB)
+
+---
+
+The Tauri Programme within the Commons Conservancy retains all rights to the modification of these trademark guidelines at any time. If you have a question or enquiry, please send an email to `trademark@tauri.app`.
+
+These guidelines are based on the Model Trademark Guidelines, available at http://www.modeltrademarkguidelines.org, used under a Creative Commons Attribution 3.0 Unported license: https://creativecommons.org/licenses/by/3.0/deed.en_EU. Version 1.0 dated 20th, August 2022
```

Comment:
not sure this here was intended, but I noted that the "version 1.0..." text on v1 is a block at the start of the page, I guess it could be at least a dedicated paragraph

Reply:
Yeah, I purposefully moved it to the end with the license information as it related to the version of the trademark guidelines and I felt it fit here more.